docs/sources/configuration/_index.md

```diff
@@ -898,6 +908,10 @@ The `frontend_worker` configures the worker - running within the Loki querier -
 # CLI flag: -querier.worker-parallelism
 [parallelism: <int> | default = 10]
 
+# Force worker concurrency to match the -querier.max-concurrent option. Overrides querier.worker-parallelism.
+# CLI flag: -querier.worker-match-max-concurrent
+[match_max_concurrent: <bool> | default = false]
```

Comment:
based on the generic placeholders section at the top, boolean values should use the placeholder <boolean>
```suggestion
[match_max_concurrent: <boolean> | default = false]
```